docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,193 @@
+What's New in Iris 2.3.0
+**********************
+
+:Release: 2.3.0
+:Date: 2019-10-04
+
+This document explains the new/changed features of Iris in version 2.3.0
+(:doc:`View all changes <index>`.)
+
+Iris 2.3.0 Features
+=================
+.. _showcase:
+
+.. admonition:: Increased Support for CF 1.7
+
+  Several changes have contributed to our increased support for CF 1.7:
+
+  * climatological coordinate support
+  * standard name modifier support
+  * name tokens enforcement in cell methods
+  * coordinate var_name, CF_CONVENTIONS_VERSION advanced to 1.7
+
+  You can read more about each of these below.
+
+.. admonition:: Climatological Coordinate Support
+
+  Iris can now load, store and save `NetCDF climatological coordinates
+  <http://cfconventions.org/Data/cf-conventions/cf-conventions-1
+  .7/cf-conventions.html#climatological-statistics>`_. Any cube time
+  coordinate can be marked as a climatological time axis using the boolean
+  property: ``climatological``. The climatological bounds are stored in the
+  coordinate's ``bounds`` property.
+
+  When an Iris climatological coordinate is saved in NetCDF, the NetCDF time
+  axis will be given a 'climatology' attribute, and the contents of the
```

Comment:
```suggestion
  coordinate variable will be given a 'climatology' attribute, and the contents of the
```
From CF conventions:

> a climatological time coordinate variable does not have a bounds attribute. Instead, it has a climatology attribute, which names a variable with dimensions (n,2), n being the dimension of the climatological time axis

So its the variable that has a `climatology` attribute

Reply:
Thanks, I appreciate the guidance. I've read and re-read that CF section but it's taking me a while to get my head around all the terminology